config/initializers/kms_encrypted_patch.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module KmsEncryptedModelPatch
+  # rubocop:disable Naming/PredicateName
+  def has_kms_key
```

Comment:
Wouldn't these need the same args? 😕

Reply:
? My intention was to not have args to this method, so that `**kms_options` isn't required for future models. 

The issue with this, though, is that a couple models DO implement version/previous version, which I've now removed... I'll need to speak to the owner of this file to confirm that's not a problem.

Are you suggesting that we should have optional args for this, so that they _can_ add args, that we ultimately ignore when prepending our version to it?

Reply:
Maybe we can leave it to a future dev to figure this out. Haha.

But my thought was that if kms_options was nil, we'd set it, otherwise, we'd set it to what we thought it should be. We could potentially merge it as well.

Reply:
ok, I'm gonna go with current implementation, considering I've confirmed with the codeowner of the form5655 files that this is fine. If someone down the road feels the need to version their records differently than the rest of the app, they can come up with a solution 😄